data_fixture/input.json

```diff
@@ -0,0 +1,6 @@
+{
+  "file_relationships": [
+    "/test_data/sample.seg"
```

Comment:
I guess this mimics what refinery will produce, but this looks like an absolute path and I could imagine that causing confusion.